R/make_table1.R

```diff
@@ -4,11 +4,12 @@
 #'
 #' @title
 #' @param data_holdout
-#' @return
-#' @author conig
+#' @return gt object with table1
+#' @author noetel
 #' @export
-make_table1 <- function(data_holdout) {
-
-  NULL
+make_table1 <- function(data_clean) {
+  # targets::tar_make()
+  # d <- targets::tar_load(data_clean)
+  names(d)
```

Comment:
I know that this is temporary, but `names(d)` would fail if you try to run the pipeline. Why not just use `data_clean` or change the input to the function to `d`?